components/DotsBackground.module.css

```diff
@@ -0,0 +1,30 @@
+.background {
+    
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    pointer-events: none;
+    z-index: -1;
+  }
+  
+  .svg {
+    width: 100%;
+    height: 100%;
+  }
+  
+  .dot {
+    fill: rgb(255, 255, 255);
+    animation: dotAnimation 5s infinite alternate ease-in-out;
```

Comment:
the animation should play randomly for each dot
if you think is too much, you can remove this line

Reply:
how frequently do you want the animation to randomly play the animation.
currently I have added the code such that at random times some circles are of opacity 1 and some of 0.4 then in a few seconds the animation changes, Will that work or should I remove the animation?

Reply:
this should be fine for now

Reply:
okay great